recipes/tensorflow-lite/all/conandata.yml

```diff
@@ -2,11 +2,21 @@ sources:
   "2.6.0":
     url: "https://github.com/tensorflow/tensorflow/archive/refs/tags/v2.6.0.tar.gz"
     sha256: "41b32eeaddcbc02b0583660bcf508469550e4cd0f86b22d2abe72dfebeacde0f"
+  "2.6.2":
+    url: "https://github.com/tensorflow/tensorflow/archive/refs/tags/v2.6.2.tar.gz"
+    sha256: "e68c1d346fc3d529653530ca346b2c62f5b31bd4fcca7ffc9c65bb39ab2f6ed3"
+  "2.9.1":
+    url: "https://github.com/tensorflow/tensorflow/archive/refs/tags/v2.9.1.tar.gz"
+    sha256: "6eaf86ead73e23988fe192da1db68f4d3828bcdd0f3a9dc195935e339c95dbdc"
 patches:
-  "2.6.0":
-    - patch_file: "patches/local_find_packages.patch"
+  "2.9.1":
+    - patch_file: "patches/msvc_fixes.patch"
```

Comment:
can you please add the extra metadata so we can track these 🙏 

https://github.com/conan-io/conan-center-index/blob/master/docs/conandata_yml_format.md#patches-fields

Reply:
Hm, when I do, I get the following error:
>[HOOK - conan-center.py] pre_export(): ERROR: [CONANDATA.YML FORMAT (KB-H030)] Additional entries ['patch_description', 'patch_type'] not allowed in 'patches':'2.9.1' of conandata.yml (https://github.com/conan-io/conan-center-index/blob/master/docs/error_knowledge_base.md#KB-H030)

Reply:
You might need to update your hooks 

https://github.com/conan-io/hooks/blob/cb2a584f66666fac92bc0602c3d43cb3b7ee8f9a/hooks/conan-center.py#L368

See https://github.com/conan-io/conan-center-index/blob/9c5c6665ee5a1800d1e4b05ff23456c1f88ccaf8/docs/how_to_add_packages.md#updating-conan-hooks-on-your-machine

Reply:
Ah, yep, that was it :)

Reply:
done!